travis/build-samples.sh

```diff
@@ -18,34 +18,50 @@ function usage() {
 
 function buildProject() {
     dir=$1
+    JHIPSTER=$dir
```

Comment:
Oh I see, your comment wasn't for this file specially but for all !
I will take a look tonight